evap/student/templates/student_vote.html

```diff
@@ -208,14 +208,18 @@ <h3 class="mb-3">{{ evaluation.full_name }} ({{ evaluation.course.semester.name
                 const lastSavedLabel = $('#last-saved');
                 if (localStorage.getItem(lastSavedStorageKey) !== null) {
                     const lastSavedDate = new Date(localStorage.getItem(lastSavedStorageKey));
-                    const delta = Math.round((timeNow - lastSavedDate) / 1000)
-                    const languageCode = "{{ LANGUAGE_CODE }}"
-                    const relativeTimeFormat = new Intl.RelativeTimeFormat(languageCode)
-                    let timeStamp = "{% trans 'unknown' %}"
+                    const delta = Math.round((timeNow - lastSavedDate) / 1000);
+                    const languageCode = "{{ LANGUAGE_CODE }}";
+                    const relativeTimeFormat = new Intl.RelativeTimeFormat(languageCode);
+                    let timeStamp;
                     if (delta < 3) {
                         timeStamp = "{% trans 'just now' %}";
+                    } else if (delta < 10) {
+                        timeStamp = "{% trans 'less than 10 seconds ago' %}";
+                    } else if (delta < 30) {
+                        timeStamp = "{% trans 'less than 30 seconds ago' %}";
                     } else if (delta < 60) {
-                        timeStamp = relativeTimeFormat.format(-delta, 'seconds');
+                        timeStamp = "{% trans 'less than 1 minute ago' %}";
```

Comment:
@janno42 "1" vs "one" -- and "10" vs "ten" above?

Reply:
numbers :)